### DISCUSSION TITLE
stored outfits inventory and defining the two fire groups

### DISCUSSION THREAD

**Discussion by daggs1** [endless-sky/endless-sky#8555] [Q&A]
Greetings,

I have two questions:
1. is there a to see which outfits are stored in which outfitter in an inventory way?
2. if I have secondary weapon on my ship, it is split between two keys, is there a way to define these fire group's content?

**Comment by mOctave** [comment]
> is there a to see which outfits are stored in which outfitter in an inventory way?

Sorry, I'm not sure I understand the question. Are you wondering whether there's a way to see which outfits are sold on which planet in an outfitter-type view (as opposed to the map view) or which outfits are included in each outfitter list as defined in `sales.txt`?

You can't look an inventory-style view of the outfitter map panel, and there is no default way to see which outfits are included in each outfitter list inventory-style, but you could write a plugin to do so (different planets with one outfitter each, sort of like Omnis but not quite).

> if I have secondary weapon on my ship, it is split between two keys, is there a way to define these fire group's content?

The two keys are for selecting the weapon you want to fire and actually firing it. It would be nice if you could set them up as fire groups, but you can't.

**Comment by lumbar527** [comment]
> 1. is there a to see which outfits are stored in which outfitter in an inventory way?

I don't think so, though you can tell if you have outfits stored on a planet in a system (Via a grey ring around the system ring in the map).
